### ISSUE TITLE
Ability to manually resolve performance alerts

### ISSUE THREAD

**Issue by ksmalik1** [getsentry/sentry#69707]
### Problem Statement

Currently, performance alert like [this](https://ender-0u.sentry.io/alerts/rules/details/239322/) one can only be auto-resolved. Auto resolved alerts create too much noise as the metric moves above or below thresholds. This is especially true for endpoints with sporadic traffic.

I can set a really low auto resolve threshold like` 0 ms` but means that the alert will never fire again once it has been triggered.

I would like the ability for:
1.  the alert to remain open until its been investigated 
2. the ability for the investigator to manually close the existing alert once their investigation is complete

Nice to have and can be treated as a separate request is the ability to add notes or assign the alert to team members for investigation. One option is for a Sentry Issue to be created anytime a performance alert fires.

### Product Area

Alerts

**Comment by leedongwei** [comment]
To paraphrase, once an alert is tripped, you want it to remain as unhealthy until someone reviews and resolves it. That seems reasonable to me. We'll keep this in mind as we work on the product. Thanks for the feedback!
